website_product_description_view/models/product_template.py

```diff
@@ -0,0 +1,25 @@
+from odoo import api, fields, models
+from odoo.tools.translate import html_translate
+
+
+class ProductTemplate(models.Model):
+    _inherit = [
+        "product.template",
+    ]
+
+    @api.depends_context("uid")
+    def _compute_can_write(self):
+        writable_templates = self._filter_access_rules("write")
+        for template in self:
+            template.can_write = template in writable_templates
+
+    website_description = fields.Html(
+        "Description for the website",
+        render_engine="qweb",
+        sanitize_attributes=False,
+        translate=html_translate,
+        sanitize_form=False,
+    )
+    can_write = fields.Boolean(
+        compute="_compute_can_write", help="The current user can edit the template."
+    )
```

Comment:
What is the purpose of these changes? We do not need this file as far as I am aware.

Reply:
I set the module to depends to base & website, thus when i didn't add this file, Odoo cannot find the website_description. but i just tried to change the dependencies to website_sale, there is no issue without this file. 
I will revise and add the improvement soon. Thank you!